src/github/pullRequestManager.ts

```diff
@@ -998,6 +1030,46 @@ export class PullRequestManager implements vscode.Disposable {
 		return;
 	}
 
+	async getSuggestedReviewers(pullRequest: PullRequestModel): Promise<SuggestedReviewer[]> {

```

Comment:
Initially implemented as separate call which will be triggered on each click on Add Reviewers.

On the other hand it is possible to get this data in [`getPullRequest`](https://github.com/microsoft/vscode-pull-request-github/blob/f39d8159f98da9cc36d8caa576ff30347f0c688b/src/github/githubRepository.ts#L318) and make this data available on `PullRequestModel`. In this way it will be fetched/updated only when user opens Description node or clicks Refresh which looks good enough.

Should I move this data to `PullRequestModel` instead of separate call?

Reply:
After implementations comparison `suggestedReviewers` on `PullRequestModel` looks better that with separate request - pushed updated `PullRequestModel`

Reply:
Nice, looks good